Examples/ArcGISToolkitExamples/AppDelegate.swift

```diff
@@ -48,6 +53,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     // Here is where we forward background fetch to the JobManager
     // so that jobs can be updated in the background
     func application(_ application: UIApplication, performFetchWithCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
+        // We only do it this way for pre-iOS 13. Otherwise we use BGTask above.
+        // This method doesn't get called for iOS 13 or later because we have an entry
+        // in the plist for BGTaskSchedulerPermittedIdentifiers.
         JobManager.shared.application(application: application, performFetchWithCompletionHandler: completionHandler)
```

Comment:
This generates a warning…sometimes. I had a screenshot of it, but I accidentally deleted it and now it isn't showing up. 😞

```suggestion
        if #available(iOS 13.0, *) {
            // Nothing to do here.
        } else {
            JobManager.shared.application(application: application, performFetchWithCompletionHandler: completionHandler)
        }
```